### ISSUE TITLE
Button not working nice on android with v2.4.1

### ISSUE THREAD

**Issue by jacargentina** [GeekyAnts/NativeBase#1749]
## react-native, react and native-base version
react-native: 0.54.4
react: 16.3.0
native-base: 2.4.1

## Expected behaviour
Display as it was on v2.3.10, just fine
## Actual behaviour
Displaying too tall
## Steps to reproduce (code snippet or screenshot)

The component source (base class)

```jsx
import React from 'react';
import { Icon, Text, H1, Button } from 'native-base';
import { View } from 'react-native';
import commonTheme from '../../native-base-theme/variables/platform';

const BaseCallToAction = props => {
  return (
    <View
      style={{
        flex: 1,
        padding: 20
      }}>
      <View
        style={{
          flex: 3,
          justifyContent: 'space-around',
          borderBottomWidth: 1,
          borderBottomColor: '#ccc'
        }}>
        <Icon
          name={props.icon}
          style={{
            fontSize: 120,
            color: commonTheme.brandPrimary,
            alignSelf: 'center'
          }}
        />
      </View>
      <H1 style={{ flex: 1, paddingTop: 20 }}>{props.title}</H1>
      <Text note style={{ flex: 2 }}>
        {props.text}
      </Text>
      <View
        style={{
          flex: 2,
          justifyContent: 'space-around'
        }}>
        <Button rounded block onPress={props.buttonHandler}>
          <Text>{props.buttonText}</Text>
        </Button>
      </View>
    </View>
  );
};

export default BaseCallToAction;

```

## Screenshot of emulator/device

![button-2 3 10](https://user-images.githubusercontent.com/517228/38264246-cde42a3c-3748-11e8-9309-ee6446037bc6.png)
**Working v2.3.10**

![button-2 4 1](https://user-images.githubusercontent.com/517228/38264271-dae98114-3748-11e8-88e6-03522e816976.png)
**Bug in v2.4.1**

## Is the bug present in both ios and android or in any one of them?
Just android; iOS works just fine (verified)
## Any other additional info which would help us debug the issue quicker.
Nop.

**Comment by SupriyaKalghatgi** [comment]
@jacargentina Please share device configuration for this Android emulator

**Comment by akhil-geekyants** [comment]
@jacargentina as a temporary fix you can set the maxHeight of button to 45(height of the button). 
We will be fixing this.

```
<Button rounded block style={{ maxHeight: 45 }}>
    <Text>CREAR UNA LISTA</Text>
</Button>
```

**Comment by jacargentina** [comment]
@SupriyaKalghatgi is this what you need? Contents of `config.ini`

```
AvdId=emulator
PlayStore.enabled=true
abi.type=x86
avd.ini.displayname=emulator
avd.ini.encoding=UTF-8
disk.dataPartition.size=800M
fastboot.forceColdBoot=no
hw.accelerometer=yes
hw.audioInput=yes
hw.battery=yes
hw.camera.back=emulated
hw.camera.front=emulated
hw.cpu.arch=x86
hw.cpu.ncore=4
hw.dPad=no
hw.device.hash2=MD5:bc5032b2a871da511332401af3ac6bb0
hw.device.manufacturer=Google
hw.device.name=Nexus 5X
hw.gps=yes
hw.gpu.enabled=yes
hw.gpu.mode=auto
hw.initialOrientation=Portrait
hw.keyboard=yes
hw.lcd.density=420
hw.lcd.height=1920
hw.lcd.width=1080
hw.mainKeys=no
hw.ramSize=1536
hw.sdCard=yes
hw.sensors.orientation=yes
hw.sensors.proximity=yes
hw.trackBall=no
image.sysdir.1=system-images/android-26/google_apis_playstore/x86/
runtime.network.latency=none
runtime.network.speed=full
sdcard.size=100M
showDeviceFrame=yes
skin.dynamic=yes
skin.name=nexus_5x
skin.path=/opt/android-sdk-linux/skins/nexus_5x
tag.display=Google Play
tag.id=google_apis_playstore
vm.heapSize=256
```

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.2)
